liquibase/changes/official_tag_update.sql

```diff
@@ -1,7 +1,9 @@
 -- Query to update official tag for feeds with contact email in the feed table where the source is mdb
 
 UPDATE public.feed f
-SET official = TRUE
+SET 
+  official = TRUE
```

Comment:
Add a comma after the first line:
```suggestion
  official = TRUE,
```